### ISSUE TITLE
Gitter chat, common BEAM channel

### ISSUE THREAD

**Issue by NobbZ** [exercism/xelixir#296]
In Exercism there are a couple of gitter-channels used to coordinate development on exercism, the product.

Also there is a global support channel and throughout exercism there are a couple of tracks that have a support channel on gitter.im. Erlang has one as well, but it isn't used quite much.

Since Erlang, Elixir, and LFE do share the same base technology (BEAM-VM) and in case of LFE even the same standard library, I'd suggest to join forces in a single gitter-channel (former erlang channel).

I can't rename the room, but already changed its headline to "Exercism Exercises in BEAM Powered Languages (Erlang, Elixir, LFE)"

If you do not want to join the channel please feel free to deny, I will remove your language from the headline. Also once joined, you are able to split into your own channel once the traffic explodes ;)

**Comment by kotp** [comment]
For reference, the [Erlang, Elixir, LFE](https://gitter.im/exercism/xerlang) gitter chat room.

**Comment by devonestes** [comment]
Well I can't see much downside to joining - what do you envision being the types of conversation happening there?

**Comment by NobbZ** [comment]
The main goal is to specialise support for installing, helping with exercises and requesting reviews for the BEAM targetting languages.

So it would extend the already existing #exercism on elixir slack to all beam related tracks on exercism, but it does not require complicated reqquesting of a slack account but only a gitter signup which happens via github-oauth-magic.

**Comment by devonestes** [comment]
Oh, so it's more for users of the product versus contributors to the underlying GitHub repos? That makes sense. At first I was just thinking "wait, don't we have Issues for this?" 😸 

So, sounds like it might be helpful to put something in `RESOURCES.md` about this since those are the intended users?

**Comment by NobbZ** [comment]
Yeah, more for the students than for coordination. It was planned for that originally but wasn't used at all and discussision happened on the issues exclusively (when there were other maintainers). So to give it some more life and meaning, I changed my mind to what it is now.

For erlang, I have linked the channel from [`docs/INSTALLATION.md`](https://github.com/exercism/xerlang/blob/master/docs/INSTALLATION.md) in a sidenote (If you have trouble join the chat) and also as a bullet point in [`docs/LEARNING.md`](https://github.com/exercism/xerlang/blob/master/docs/LEARNING.md).

I do hope students will find those links once Katrina has deployed the latest version.
